src/_gettsim/functions_loader.py

```diff
@@ -310,7 +340,8 @@ def _load_aggregation_combined_dict_from_strings(sources):
             aggregation_dicts_defined_in_module = [
```

Comment:
```suggestion
            relevant_dicts_defined_in_module = [
```

But maybe the better option is to just do two dictionaries? I find the lambda functions below pretty confusing. 

Or add an argument to the function whether we want aggregation or interpersonal links? Probably best, unless 
1. it is too expensive to do the imports twice or 
2. we need both dicts at the same time for the consistency checks.

Seems to me it can't be 1. as we are calling it multiple times and discarding one of the outputs.